test/unit/setup/gutenberg-phase.js

```diff
@@ -0,0 +1,5 @@
+process.env = {
```

Comment:
@gziolo I struggled to find a way to inject the `GUTENBERG_PHASE` environment variable for unit tests, I couldn't handle it in the npm script like I do with webpack. I came up with this, would be great to get your thoughts on it. Maybe you're aware of a better way.

Reply:
That's very strange. I'm personally fine with such workaround. As long as it works 🤷‍♂️

Reply:
Unusually `process.env.GUTENBERG_PHASE = 2` doesn't seem to work. I wish I had a good explanation.

Reply:
Can we add
```js
// Inject the `GUTENBERG_PHASE` global, used for feature flagging.
```
also here?

Reply:
👍 Done.